frontend/src/component/pages/shared/Header.tsx

```diff
@@ -13,9 +14,9 @@ export class Header extends Component<Props> {
         <div className={'center'}>
           <div id="logo">Short</div>
           <div id="spacer"></div>
-          <div className={'sign-out'}>
+          {this.props.shouldShowSignOutButton && <div className={'sign-out'}>
```

Comment:
Can we move the <div> to its own line? Just trying to make it easy to read.

Reply:
{this.props.shouldShowSignOutButton && (
      <div className={"sign-out"}>
        <Button onClick={this.props.onSignOutButtonClick}>Sign out</Button>
      </div>
    )}

Reply:
Yes!